charts/keycloakx/README.md

```diff
@@ -0,0 +1,628 @@
+# Keycloak-X
+
+[Keycloak-X](http://www.keycloak.org/) is an open source identity and access management for modern applications and services.
+
+## TL;DR;
+
+```console
+$ helm install keycloak codecentric/keycloakx
+```
+
+## Introduction
+
+This chart bootstraps a [Keycloak](http://www.keycloak.org/) StatefulSet on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+It provisions a fully featured Keycloak installation.
+For more information on Keycloak and its capabilities, see its [documentation](http://www.keycloak.org/documentation.html).
+
+## Prerequisites Details
+
+The chart has an optional dependency on the [PostgreSQL](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) chart.
+By default, the PostgreSQL chart requires PV support on underlying infrastructure (may be disabled).
+
+**Important**
+Since Version `v1.0.0` the support for Helm v2 is dropped in favor of to support also the latest postgres Helm Chart Version which requires Helm v3 only.
```

Comment:
This paragraph is probably not needed in this chart as it did never support Helm v2, right?